services/api/src/emailSender.py

```diff
@@ -9,31 +12,45 @@ def __init__(self, smtp_server, port):
         self.context = ssl._create_unverified_context()
         self.server = smtplib.SMTP(self.smtp_server, self.port)
 
-    def send(self, sender, recipient, subject, message, replyEmail, username, password):
+    def send(
+        self,
+        sender,
+        recipient,
+        subject,
+        message,
+        replyEmail,
+        tlsEnabled,
+        authEnabled,
+        username,
+        password
+    ):
         try:
-            self.server.ehlo()  # say hello to server
-            self.server.starttls(context=self.context)  # start TLS encryption
-            self.server.ehlo()  # say hello again
-            self.server.login(username, password)
-
             # prepare email
-            toSend = self.prepareEmailToSend(
+            toSend = ""
+            toSend = self.format(
                 sender, recipient, subject, message, replyEmail
             )
+            if tlsEnabled == "true":
+                self.server.starttls(context=self.context)  # start TLS encryption
+                self.server.ehlo()  # say hello
```

Comment:
Do we not need to ehlo for non-tls systems?

Reply:
Apparently this function is implicitly called by .sendmail() if needed, so we do not need to explicitly call it here.

Source: https://realpython.com/python-send-email/#option-2-using-starttls